packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -92,7 +93,8 @@ export type RouteOptions =
     | ITinyliciousRouteOptions
     | IOdspRouteOptions;
 
-function wrapWithRuntimeFactoryIfNeeded(packageJson: IFluidPackage, fluidModule: IFluidModule): IFluidModule {
+async function wrapWithRuntimeFactoryIfNeeded(packageJson: IFluidPackage, fluidModule: IFluidModule):
```

Comment:
This seems like it should stay synchronous?

Reply:
I do not see added async call that would need to make this call return a promise.

Reply:
nvm...it should be synchronous